### ISSUE TITLE
Keyboard Layout indicator

### ISSUE THREAD

**Issue by EdwinvRooij** [budgie-desktop/budgie-desktop#866]
**Operating System**: Solus 2017.04.18.0 64-bit
**Budgie version**: 10.3.1

Not a bug, but a feature request/enhancement.

I find myself wondering what my current keyboard language is on quite frequently when trying to login. I'm an English/Dutch Dvorak typist since a month. It is not clear to me if I should type my password using a Dvorak layout or Qwerty. I can not seem to change this eiter, from the login page.

When I change it using 'win + space', a visual indicator would be a great asset to me, since I switch between layouts on a daily basis, because of some applications (shortcuts designed for Qwerty) and at startup, like I said, when logging in. 

tl;dr: visual indicator for keyboard layout switching and enabling switching (or set default) on login

PS. The new alt tab looks and works great!

**Comment by niyasc** [comment]
Where do you exactly experience this issue? In login screen or in lock screen?

If it is just on lock screen, there is a similar issue reported here. https://github.com/budgie-desktop/budgie-desktop/issues/788

In login screen, there is a language indicator on top of screen, at least, is present in **Ubuntu Budgie** and on **Solus 1.2**. Need to confirm on **Solus (current)**.
![image](https://cloud.githubusercontent.com/assets/2451833/25128638/f7a79cfa-2457-11e7-99d2-a2d7cc09c9f3.png)
*budgie-remix 16.04.x login screen*

![image](https://cloud.githubusercontent.com/assets/2451833/25128727/692f6042-2458-11e7-8c4c-50398d1628bc.png)
*Solus 1.2 login screen*

**Comment by EdwinvRooij** [comment]
Maybe I should have created two seperate issues for this. 
I'm talking about the login screen, but I suppose it will be worked on in a similar manner to the lock screen, so let's leave that one to the issue you just referenced.

This is the login screen. There is indeed a language indicator, but I'm still unable to change it.
![img_20170418_202142](https://cloud.githubusercontent.com/assets/16732313/25146721/3f89ed3a-2475-11e7-8f68-ace1c8804227.jpg)

As for the enhancement, a simple UI indicator to show to which layout you're switching would be great, but probably not a high priority right now.

**Comment by JoshStrobl** [comment]
Duplicate of #788

**Comment by EdwinVanRooij** [comment]
I may have not been completely clear, but it seems the problem I was facing is easily solved. Posting this for future reference:

You can change the keyboard layout at login screen for __all users__:

- Go to 'Settings' (shortcut: alt + f2, type 'settings', select respectively)
- Go to 'Region & Language'
- Press the 'Login Screen' __button__ at the __top-right corner__
- Change the layout as desired in the box below the 'Input Sources'

I would've appreciated this information if I knew it earlier, so I hope I'm helping someone out by providing it now.
